INSTALL.linux.rst

```diff
@@ -0,0 +1,123 @@
+=====
+Linux
+=====
+
+
+  Tested on Arch Linux, Linux Mint 20, Ubuntu 20.04 and Fedora 33 (all amd64)
+
+
+Build Requirements
+------------------
+
+- cmake
+- make
+- ninja
+- gcc
+- g++
+- nasm
+- git
+- sudo
+- python (3.7+)
+- python development (python3-dev or python3-devel)
+- pip
+
+
+Arch
+^^^^
+
+.. code-block:: sh
+
+  sudo pacman -Sy
+  sudo pacman -S cmake make ninja gcc nasm git python3 python-pip
+
+
+Fedora
+^^^^^^
+
+.. code-block:: sh
+
+  sudo dnf update
+  sudo dnf install cmake make ninja-build gcc g++ nasm git python3-devel python3-pip
+
+
+Ubuntu
+^^^^^^
+
+.. code-block:: sh
+
+  sudo apt-get update
+  sudo apt-get install cmake make ninja-build gcc g++ nasm git python3-dev python3-pip
+
+
+Build Steps
+-----------
+
+#. Install cffi module from pypi:
+
+   - ``sudo pip3 install --upgrade cffi``
```

Comment:
You should avoid running `pip` with `sudo` (which will apply again below).

But specifically for `cffi`, `avif` declares it depends on CFFI already, so you shouldn't have to install it manually at all, it'll be installed automatically, *except* if you're documenting how to run rebuilding the shared library during development, i.e. running `python avif/_build.py`? Then yes you need a virtualenv where either avif has been installed or at least cffi is present.